scripts/add_electricity.py

```diff
@@ -218,7 +223,17 @@ def load_powerplants(ppl_fn=None):
         columns=["efficiency"]).replace({"carrier": carrier_dict}))
 
 
-def attach_load(n, regions, load, admin_shapes, countries, scale):
+def attach_load(
```

Comment:
You are now having a function which requires each of the variables as input. This means they must assign the variables before you need them at the execution step.


```
def attach_load(
    n,
    regions,
    weather_year,
    prediction_year,
    region_load,
    ssp,
    admin_shapes,
    countries,
    scale,
):
```